crypto/x509/by_dir.c

```diff
@@ -379,7 +389,7 @@ static int get_cert_by_subject(X509_LOOKUP *xl, X509_LOOKUP_TYPE type,
              * If we were going to up the reference count, we would need to
              * do it on a perl 'type' basis
              */
-        /*- CRYPTO_add(&tmp->data.x509->references,1,
+        /*- CRYPTO_add(&tmp->data.x509->references, 1,
```

Comment:
This dead code and the preceding comment are *really* old and misaligned. Instead of fixing line 392, I would suggest to just remove lines 389-393 entirely. My suggestion would be to squash 619cf90fcb65 and 6849dea554a1 into a single commit, but without the change here in line 392.

If you like, you can add a second commit that removes the old unused code (lines 389-393), or you do it in a separate PR.

Reply:
I noticed it, but edit it by mistake;  it can stay as it is, for a separate PR.